### PR TITLE
PMM-6338 Final switch to MkDocs/md

### PR DESCRIPTION
**Pull request by PaulJacobs-percona**
Renamed source to .source. Will consider deleting after a bedding-in period for live publishing of MkDocs content.

Associated Jenkins job for publishing is now https://pmm.cd.percona.com/job/pmm-2-x-documentation-mkdocs/

IMPORTANT: RST files no longer kept in sync from this point on.

### CODE REVIEW

**Comment by rnovikovP** [comment]
what the reason to keep it if we'll not use and will not update? 
We can tag this commit and delete files. If we'll ever need them - get this tagged commit and   restore them

File: .source/percona-theme/changes/frameset.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Frameset//EN"
+  "http://www.w3.org/TR/html4/frameset.dtd">
+<html>
+  <head>
+    <title>{% trans version=version|e, docstitle=docstitle|e %}Changes in Version {{ version }} &mdash; {{ docstitle }}{% endtrans %}</title>
+  </head>
+  <frameset cols="45%,*">
+    <frame name="main" src="changes.html">
+    <frame name="src" src="about:blank">
```
Comment (atymchuk) [inline .source/percona-theme/changes/frameset.html:9]: @PaulJacobs-percona Just a sanity check: do we really need that second frame with no content? On purpose I guess?

File: README.md [hunk 2]
```diff
@@ -65,40 +56,17 @@ There are three ways to get changes made to the documentation. Two are 'do it yo
...
-If you have [Docker installed](https://docs.docker.com/get-docker/), a more convenient way is to use our Docker image as follows:
-
-1. Clone this repository.
-
-2. `cd pmm-doc`
-
-3. `docker run --rm -v $(pwd):/docs perconalab/pmm-doc-md`
-
-4. Open `site/index.html` in a browser to view the first page of documentation.
+5. Open `site/index.html` in a browser to view the first page of documentation.
```
Comment (atymchuk) [inline README.md:69]: For clarity sake, I suggest the provide an url
```suggestion
5. Open `https://localhost/site/index.html` in a browser to view the first page of documentation.
```

**Review by atymchuk — CHANGES_REQUESTED** [review]

File: .source/percona-theme/changes/frameset.html (hunk 1 above)
Comment (PaulJacobs-percona) [reply .source/percona-theme/changes/frameset.html:9]: This is part of the to-be-deprecated RST source. And percona-theme is downloaded fresh each build by the makefile and used by all Sphinx projects in Percona (afaik). In other words, I should have said 'ignore everything under .source).

File: README.md (hunk 2 above)
Comment (PaulJacobs-percona) [reply README.md:69]: It's a static file on the local filesystem. It's not being served by a web server (as it is when running `mkdocs serve`).

File: .source/percona-theme/changes/frameset.html (hunk 1 above)
Comment (atymchuk) [reply .source/percona-theme/changes/frameset.html:9]: Oh, I see. I'd be great to leave an comment `<-- ^^ -->` there :)

File: README.md (hunk 2 above)
Comment (atymchuk) [reply README.md:69]: Ok, makes sense.

**Review by atymchuk — APPROVED** [review]

**Comment by PaulJacobs-percona** [comment]
> what the reason to keep it if we'll not use and will not update?
> We can tag this commit and delete files. If we'll ever need them - get this tagged commit and restore them

For the first few days/weeks, I expect questions to arise about the accuracy of the migration. Keeping a (hidden) copy of the original RST makes it easier to check and validate the migrated material.

**Review by shoffman-percona — APPROVED** [review]
Not sure what comment I can leave, did not read the code line by line but the gist of the move seems to make sense.

**Review by rnovikovP — APPROVED** [review]
/me adding a reminder to check it's  removed in 2 weeks )

**Pull request merged**